src/page/ConfigPageLayout/tabs/SecretsManagementTab/SecretsManagementUI.tsx

```diff
@@ -81,7 +81,7 @@ export function SecretsManagementUI() {
       )}
 
       {editMode && (
-        <SecretEditModal id={editMode} existingNames={existingNames} open onDismiss={() => handleEditSecret()} />
+        <SecretEditModal name={editMode} existingNames={existingNames} open onDismiss={() => handleEditSecret()} />
```

Comment:
this is one where I'm scratching my head. I don't understand if this `id` here is supposed to be an id parameter, or some kind of id attribute that has nothing to do with the edit operation.